src/main/java/io/gravitee/repository/mongodb/management/internal/plan/SubscriptionMongoRepositoryImpl.java

```diff
@@ -82,6 +82,22 @@
             query.addCriteria(Criteria.where("updatedAt").gte(new Date(criteria.getFrom())).lt(new Date(criteria.getTo())));
         }
 
+        if (criteria.getEndingAtAfter() > 0 || criteria.getEndingAtBefore() > 0) {
+
+            // Need to mutualize the instantiation of this criteria otherwise mongo drive is throwing an error, when
+            // using multiple `Criteria.where("endingAt").xxx` with the same query
+            Criteria endingAtCriteria = Criteria.where("endingAt");
+
+            if (criteria.getEndingAtAfter() > 0) {
+                endingAtCriteria = endingAtCriteria.gte(new Date(criteria.getEndingAtAfter()));
+            }
+            if (criteria.getEndingAtBefore() > 0) {
+                endingAtCriteria = endingAtCriteria.lt(new Date(criteria.getEndingAtBefore()));
```

Comment:
In mongoDB, we use a "<=" (less than or equal).
Should we use a `lte` here, or change the mongoDB implementation ?